test/test_ops.py

```diff
@@ -339,11 +350,45 @@ def test_out(self, device, dtype, op):
         sample = samples[0]
         # call it normally to get the expected result
         expected = op(*sample.input, *sample.args, **sample.kwargs)
-        # call it with out=... and check we get the expected result
-        out_kwargs = sample.kwargs.copy()
-        out_kwargs['out'] = out = torch.empty_like(expected)
-        op(*sample.input, *sample.args, **out_kwargs)
-        self.assertEqual(expected, out)
+
+        def _test(tested_op):
+            # call it with out=... and check we get the expected result
+            out_kwargs = sample.kwargs.copy()
+            out_kwargs['out'] = out = torch.empty_like(expected)
+            tested_op(*sample.input, *sample.args, **out_kwargs)
+            self.assertEqual(expected, out)
+
+        _test(op)
+        for a_op in op.aliases:
+            _test(a_op)
+
+    @ops([op for op in op_db if op.aliases])
+    def test_aliases_jit_name_remapping(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype, requires_grad=True)
+        if len(samples) == 0:
+            self.skipTest("Skipped! No sample inputs!")
+
+        # NOTE: only tests on first sample
+        sample = samples[0]
+
+        sample_args_kwargs = ()
+        if len(sample.args) > 0:
+            sample_args_kwargs += (sample.args, )
+        if len(sample.kwargs) > 0:
+            sample_args_kwargs += (sample.kwargs, )
+
+        for a_op in op.aliases:
+
+            def _fn(*sample_args, **sample_kwargs):                
```

Comment:
This is cool but it's only testing TRACING, not SCRIPTING. See my modified test above, derived from the one in test_op_aliases.py. Basically we want to copy that test and update its data source to use OpInfos instead of its current AliasInfos.